Collaborative Robot.md

```diff
@@ -0,0 +1,11 @@
+---
+aliases: cobot
+tags: industrial robot UR
```

Comment:
these tags are all interpreted, no comma necessary. However, this means we currently have the tags _robot_, _robotics_ and _Robotic_, not very clear.. you can enable the core plugin _Tag Pane_ in Obsidian to get an overview and sort this a little bit..